.circleci/config.yml

```diff
@@ -78,14 +78,30 @@ commands:
     steps:
       - run:
           name: publish m2 and local
-          command: sbt publishM2 publishLocal
+          command: sbt publishM2 publishLocal -Ddisable.apidocs=true
```

Comment:
That's a good one. Doc generation is incredible slow. That is probably what takes most time of the doc Make.